src/app/main/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.ts

```diff
@@ -11,6 +11,7 @@ import { HabitAssignInterface } from '@global-user/components/habit/models/inter
 import { FriendProfilePicturesArrayModel } from '@global-user/models/friend.model';
 import { MatDialog } from '@angular/material/dialog';
 import { FriendsListPopUpComponent } from '@global-user/components/shared/components/friends-list-pop-up/friends-list-pop-up.component';
+import { habitImages } from '../../../../../../image-pathes/habits-images';
```

Comment:
Not an issue just prefer non-relative pathes for imports to avoid ../ (Can be configured in IDE)